src/locales/de/translation.json

```diff
@@ -81,6 +81,7 @@
   "Close modal": "Modale Fenster schließen",
   "Debug": "Debug",
   "Options": "Optionen",
+  "<0>Open in OSM</0> &mdash; Opens the current view on openstreetmap.org": "0>Öffnen in OSM</0> &mdash; Öffnet die aktuelle Ansicht auf openstreetmap.org",
```

Comment:
These should be resolved, it's a bug in i18next I think...

Reply:
Ops! I forgot a  "<"